src/Tmds.Ssh/MLKem768X25519Sha256KeyExchange.cs

```diff
@@ -0,0 +1,122 @@
+﻿// This file is part of Tmds.Ssh which is released under MIT.
+// See file LICENSE for full license details.
+
+using System.Buffers;
+using System.Security.Cryptography;
+using Microsoft.Extensions.Logging;
+using Org.BouncyCastle.Crypto;
+using Org.BouncyCastle.Crypto.Agreement;
+using Org.BouncyCastle.Crypto.Generators;
+using Org.BouncyCastle.Crypto.Kems;
+using Org.BouncyCastle.Crypto.Parameters;
+using Org.BouncyCastle.Crypto.Prng;
+using Org.BouncyCastle.Security;
+
+namespace Tmds.Ssh;
+
+// Post-Quantum Traditional (PQ/T) Hybrid key exchange methods using ML-KEM-768 and X25519 with SHA-256
+// https://www.ietf.org/archive/id/draft-kampanakis-curdle-ssh-pq-ke-05.html
+sealed class MLKem768X25519Sha256KeyExchange : Curve25519Sha256KeyExchange
```

Comment:
Let's also remove the hash name from the class name, and apply the same to `SNtruPrime761X25519Sha512KeyExchange`.

The rationale for not including the hash, is that if we'd add variations based on the hash, we'd add an argument to this class to pass the different hash algorithms.